README.md

```diff
@@ -114,6 +124,20 @@ The **web app** runs on any device and is mobile optimised. It provides:
 
 Adding push buttons to control volume, skipping tracks, pause, play: read the [GPIO buttons installation guide](docs/GPIO-BUTTONS.md).
```

Comment:
Would it be better to move the GPIO doc to the wiki and link to the wiki page?

Reply:
Yes, we can do that.
But I think the wiki needs some editing, too. E.g. many of manual steps described aren't necessary anymore with the install script.